src/DssSpell.sol

```diff
@@ -18,37 +18,33 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import { VatAbstract } from "dss-interfaces/dss/VatAbstract.sol";
-import { VestAbstract } from "dss-interfaces/dss/VestAbstract.sol";
 import { GemAbstract } from "dss-interfaces/ERC/GemAbstract.sol";
 
-interface RwaLiquidationOracleLike {
-    function bump(bytes32 ilk, uint256 val) external;
-}
-
 interface ProxyLike {
     function exec(address target, bytes calldata args) external payable returns (bytes memory out);
 }
 
+interface RwaLiquidationLike {
+    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
+}
+
+interface VestLike {
+    function yank(uint256 _id) external;
+}
```

Comment:
Although not being an error, I would suggest to import `VestAbstract` from `dss-exec-lib` instead of redefining VestLike interface

Reply:
As we use only one method from Ves contract it's a rule to define those instead of import (see archive)